src/main/scala/sbtdocker/DockerPush.scala

```diff
@@ -44,5 +47,20 @@ object DockerPush {
     val process = Process(command)
     val exitValue = process ! processLog
     if (exitValue != 0) sys.error("Failed to push")
+
+    val PushedImageId = ".* sha256:([0-9a-f]+) .*".r
```

Comment:
I think we should be stricter in this matching, in case the output is changed at a later stage:

```suggestion
    val PushedImageId = ".* digest: sha256:([0-9a-f]+) .*".r
```